sdk/eventhub/event-hubs/samples/tsconfig.json

```diff
@@ -1,13 +1,35 @@
 {
-  "extends": "../tsconfig.json",
   "compilerOptions": {
-    "module": "commonjs"
+    /* Basic Options */
+    "target": "es2016" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
```

Comment:
Was `es2016` chosen because it runs on all versions of node.js we support? (I'm not 100% sure on that, just an assumption).

`es2018` would be amazing if we only needed to support node.js 10+ (node 8.x should be end of life this month) since then we wouldn't need to transpile async/await, but if we need node 8 support then this seems fine to me.

Reply:
It wasn't a conscious choice, it just seemed to be a common target.

@willmtemple  and I were talking about using your es2018 trick to possibly generate the JS samples since the output is _really_ clean.